### ISSUE TITLE
This library is still supported?

### ISSUE THREAD

**Issue by zion03** [Vestorly/torii#461]
Hello there, а little bit sad that this awesome lib in unmaintained state, anybody has access to this lib? 
There a few significant commit in master but not tagged. 
The last tag was made on Feb 6, 2018

**Comment by natbatwat** [comment]
Wondering the same thing

**Comment by Unkrass** [comment]
@rwwagner90 is willing to continue maintaining this repo. He will probably fork and continue with the forked repo as commented here: https://github.com/Vestorly/torii/issues/424#issuecomment-769361374

**Comment by rwwagner90** [comment]
@mixonic and @Dhaulagiri have access I believe. Do either of you know if there is an appetite to add more maintainers or transfer to adopted-ember-addons before we fork this?

**Comment by jayvarner** [comment]
@rwwagner90 transferring to adopted-ember-addons seems appropriate. I'm willing to help.

**Comment by rwwagner90** [comment]
@jayvarner do you know anyone with the permissions to transfer it?

**Comment by snewcomer** [comment]
hi!  Ping on this.

specifically, ember v4.0 will not work here - https://github.com/Vestorly/torii/blob/master/addon/bootstrap/session.js#L8

**Comment by rwwagner90** [comment]
@Dhaulagiri @jayvarner does anyone have the permissions to transfer this repo please? Otherwise, I suppose we could fork it to adopted-ember-addons.

**Comment by jayvarner** [comment]
No response here for 6+ months, last merged PR 2+ years ago...fork it.

**Comment by Dhaulagiri** [comment]
I wish I could be more helpful here but I only have commit bit on github and nothing else that would unblock y'all. I would definitely be supportive of transferring this to ember-adopted-addons but that's something @mixonic would have to help with (I think).

**Comment by halfbyte** [comment]
@rwwagner90 Did you try to contact @mixonic directly? Would you mind if I shoot him an email? 

I am very interested in getting this back into a maintained state and am pretty sure I can support an effort to modernise it with some time from our team. We're very close to forking it ourselves as it definitely starts to rattle and crack in a more modern ember app. It somehow, with a couple of ugly workarounds, still works, but barely.

**Comment by rwwagner90** [comment]
@halfbyte yeah, I have tried reaching out to @mixonic, but I don't think he (or anyone for that matter) has the rights to transfer the repo or give us access. I tried to open a necessary PR and it sat for over a year and no one would merge it, so I think we should just fork. We could fork it into adopted-ember-addons perhaps, instead of trying to wait for someone to actually transfer it.

**Comment by halfbyte** [comment]
@rwwagner90 Is there any way I can help? We have a bunch of fixes lying around already that could go into a fork, I think we just need your help for actually creating the fork and then we can open PRs on the new repo - Happy to share some maintainer duties as well if possible (and of course, if you're happy with our work)

**Comment by rwwagner90** [comment]
@halfbyte I created the fork and gave you write access. If you can port over some of the PRs here, that would be great! https://github.com/adopted-ember-addons/torii

**Comment by halfbyte** [comment]
@rwwagner90 Thanks so much. Would you have time to do some PR reviews or should we keep you out of this as much as possible? I assume you're as busy as the rest of us, so don't want to put an extra burden on you if you think it's not necessary. 

My idea right now is to maybe amass changes that could lead to an LTS + 4.x compatible 1.0.0 on a branch and collect feedback on that before thinking about an actual release. (Then we would need to talk about where and how to publish the fork, but that's something to worry about when we get there)

**Comment by rwwagner90** [comment]
@halfbyte I'm generally okay with trusting your judgement. Do you have some other folks on your team that can help review? Perhaps we should rope in @snewcomer and @Dhaulagiri?

**Comment by halfbyte** [comment]
@rwwagner90 Cool, that sounds good, I have some teammates that will look at it, @lozjackson actually did most of the relevant changes on our fork, so then I'll review - I'll get back to y'all if I have thorny issues or if we are insecure about something!

Again, thanks for the quick response and your trust.

**Comment by halfbyte** [comment]
In case anyone is wondering, we're close to releasing a new version over at adopted-ember-plugins/torii so I would be happy for anyone testing the current master branch over there.

**Comment by halfbyte** [comment]
Finally, a 1.0.0-beta.1 has been released to npm. Please test and head over to [adopted-ember-plugins/torii] to report any issues.

**Comment by rwwagner90** [comment]
https://github.com/adopted-ember-addons/torii
